docs/Reference/Reply.md

```diff
@@ -84,7 +84,6 @@ object that exposes the following functions and properties:
   from Node core.
 - `.log` - The logger instance of the incoming request.
 - `.request` - The incoming request.
-- `.context` - Access the [Request's context](./Request.md) property.
```

Comment:
we should kepp it right now

Reply:
Also done in [f3be7b5](https://github.com/fastify/fastify/pull/4216/commits/f3be7b547b7708e104136c93ca3140f32eabd241)